js-intermediate.js

```diff
@@ -4,8 +4,29 @@
 // Be sure to use all given test cases
 
 // 1. Write a function called getFib that returns the first 10 numbers of the Fibonacci sequence in an array. Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
-
-
+//longform
+const getFib = (n) => {
+  let fibArray = []
+  let placeholder = 0 
+  for(let i=0; i<=n; i++){
+    placeholder += i
+    fibArray.push(placeholder)
+  }
+  return fibArray
+}
+console.log(`#1: ${getFib(10)}`)
+//refactor- wanted to use map but researched and realized not a great solution; came across a recursive solution which I like so I stole; <https://medium.com/quick-code/fibonacci-sequence-javascript-interview-question-iterative-and-recursive-solutions-6a0346d24053>
+// below totally doesn't work yet but later id like to keep going down this rabbit trail
+// function fib(n) {
+//   let fibArray = []
+//   if (n < 2){
+//     fibArray.push(n)
+//   } else {
+//     fibArray.push(fib(n - 1) + fib (n - 2))
+//   }
+//   return fibArray
+// }
+// console.log(fib(10))
 
```

Comment:
This is a tricky problem for sure. Keep working on it in your spare time. I can see this being an interesting point of conversation in a tech interview.